src/paho/mqtt/client.py

```diff
@@ -222,7 +321,9 @@ def connack_string(connack_code):
         return "Connection Refused: unknown reason."
 
 
-def base62(num, base=string.digits + string.ascii_letters, padding=1):
+def base62(
+    num: int, base: str = string.digits + string.ascii_letters, padding: int = 1
```

Comment:
I'd add the trailing comma already, so this would reformat to
```suggestion
    num: int,
    base: str = string.digits + string.ascii_letters,
    padding: int = 1,
```